src/lib/sidebar/EntireScheme.svelte

```diff
@@ -36,23 +44,28 @@
       try {
         let resp = await fetch(loadUrl);
         let body = await resp.text();
-        gjScheme.set(backfill(JSON.parse(body)));
+        gjSchemeCollection.set(backfill(JSON.parse(body)));
       } catch (err) {
         console.log(`Failed to load from URL: ${err}`);
       }
     } else if (loadLocal) {
       try {
-        gjScheme.set(backfill(JSON.parse(loadLocal)));
+        gjSchemeCollection.set(backfill(JSON.parse(loadLocal)));
       } catch (err) {
         console.log(`Failed to load from local storage: ${err}`);
       }
     }
     loaded = true;
+    scheme = getFirstSchemeOrEmptyScheme($gjSchemeCollection);
```

Comment:
It feels like a few places set up this default empty scheme if it's missing. Is it something `backfill` can do? We always call that when loading a file from anywhere. It should return a valid SchemeCollection. Part of that could be filling out this default empty bit